lib/services/datasources/local_datasource.dart

```diff
@@ -129,12 +174,34 @@ class LocalDataSourceImpl implements LocalDataSource {
   }
 
   @override
-  Future<bool> saveNft(NFT draft) async {
+  Future<int> saveNft(NFT draft) async {
+    try {
+      final result = await database.nftDao.insertNft(draft);
+      return result;
+    } catch (e) {
+      throw "";
```

Comment:
this can be rewritten since exception thrown is an empty string in multiple try/catch statements.